src/app/(test)/jin/page.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { FloatingButton } from '../../../components/Floating/FloatingButton';
+
+const generateContent = (count: number) => {
+  const content = [];
+  for (let i = 1; i <= count; i += 1) {
+    content.push(<p key={i}>{i}</p>);
+  }
+  return content;
+};
+
+export default function HomeApp() {
+  return (
+    <div>
+      <div className="p-4">{generateContent(100)}</div>
```

Comment:
```suggestion
      <div className="h-[100rem]">아무글자</div>
```
단순히 화면을 늘려서 테스트 싶은거라면 요런 방법도 있습니당~